tools/test-tools

```diff
@@ -25,9 +25,7 @@ if __name__ == "__main__":
     if args.coverage:
         import coverage
 
-        cov = coverage.Coverage(
-            branch=True, omit=["*/zulip-venv-cache/*", os.path.join(tools_test_dir, "*")]
-        )
+        cov = coverage.Coverage(branch=True, omit=[os.path.join(tools_test_dir, "*")])
```

Comment:
Hmm, why does the new venv path not need to be passed here?

Reply:
Our odd symlink setup had been preventing Coverage.py from detecting the virtualenv as third-party; now it does so correctly, as can be seen with `debug=["trace"]`.